CONTRIBUTING.md

```diff
@@ -44,6 +44,12 @@ the code. Please attempt to match the style of surrounding code as much as possi
 components, prefer the patterns described in the [C++ core guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines)
 and the [modern C++/WinRT language projections](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/).
 
+The projects in this codebase typically make use of precompiled headers (pch.h). When it is necessary to include a
```

Comment:
If someone doesn't understand this information the consequences are not very severe. Either they'll add a header only to the pch, in which case we'll catch it when we have automated Clang builds set up. Or they'll add it only to the specific file, in which case there's no harm, just a missed perf improvement.

So...can this just be a comment in pch.h, to keep CONTRIBUTING from getting bloated with obscure details that most people don't need to care about?

Reply:
I see your point about it feeling out-of-place here and I'm fine with moving the comment.  @HowardWolosky, thoughts?

Reply:
Move to the CalcManager pch.